src/server.rs

```diff
@@ -74,10 +74,11 @@ impl Server {
                 .or(warp::patch()
                     .and(store.clone())
                     .and(api_kv_key_path)
-                    .and(filters::body::content_length_limit(
-                        configuration.http.request_size_limit,
-                    ))
```

Comment:
Why remove the request size limit for the PATCH method? The server expects a request body here, so there should be a limit on the size.

Reply:
Yes, you right, I was sure that it was a mistake due to copy/paste, I will add it!

Reply:
In my sense patch request will be always short its the reason why I've removed this but it's an error